### ISSUE TITLE
Warning to install ruby in Mac OS X 10.5.8(PowerPC)

### ISSUE THREAD

**Issue by ywjno** [rbenv/ruby-build#656]
When I run command `rbenv install 2.1.4` in my macmini(Mac OS X 10.5.8, PowerPC machine), I has this warning

```
-> % rbenv install 2.1.4
unknown option '-sha256'
options are
-c              to output the digest with separating colons
-d              to output debug info
-hex            output as hex dump
-binary         output in binary form
-sign   file    sign digest using private key in file
-verify file    verify a signature using public key in file
-prverify file  verify a signature using private key in file
-keyform arg    key file format (PEM or ENGINE)
-signature file signature to verify
-binary         output in binary form
-engine e       use engine e, possibly a hardware device.
-md5 to use the md5 message digest algorithm (default)
-md4 to use the md4 message digest algorithm
-md2 to use the md2 message digest algorithm
-sha1 to use the sha1 message digest algorithm
-sha to use the sha message digest algorithm
-mdc2 to use the mdc2 message digest algorithm
-ripemd160 to use the ripemd160 message digest algorithm
Downloading ruby-2.1.4.tar.gz...
```

But the ruby has installed success.

The `rbenv` and `ruby-build` source has lastest.

**Comment by Ole2mail** [comment]
homebrew itself is a problem to install as 10.5.8 has outdated version of ruby and so it's failing on installation scripts.

**Comment by barracuda156** [comment]
> The OpenSSL version shipped with OS X Leopard (10.5) doesn't support SHA-256. A workaround is to install a newer version of OpenSSL and put it in the `$PATH` (although might cause issues), or to install e.g. GNU coreutils via Homebrew. Instead of forcing users to symlink `gsha256sum` to `sha256sum` might consider extending the detection algorithm to look for `gsha256sum`?

Macports has the current `openssl`, and all its versions build successfully for `ppc` on 10.4.11, 10.5.8, 10.6 PPC and 10.6.8 and for `ppc64` on 10.5.8.
